frontend/src/components/MeetingRoom/ChatInput/ChatInput.tsx

```diff
@@ -26,12 +31,22 @@ const ChatInput = (): ReactElement => {
 
   const handleKeyDown = (e: KeyboardEvent<HTMLDivElement>) => {
     // If enter press then send message unless shift also pressed to allow for multiline messages
-    if (e.key === 'Enter' && !e.shiftKey) {
+    if (e.key === 'Enter' && !e.shiftKey && !isComposing) {
       e.preventDefault();
       handleSendMessage();
     }
   };
 
+  const handleCompositionStart = () => {
+    setIsComposing(true);
+  };
+
+  const handleCompositionEnd = () => {
+    setIsComposing(false);
+    // Ensure text is updated after composition ends
+    setText((prevText) => prevText);
```

Comment:
I think we can remove these two lines, setting state to itself has no effect, I tested this and saw no issue with removing this line